libs/common/pch.h

```diff
@@ -0,0 +1,12 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+
+#ifndef PCH_H
+#define PCH_H
+
+// add headers that you want to pre-compile here
+#include "framework.h"
```

Comment:
Since there's only one, can we get rid of pch.h?

Reply:
removed.